Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    user
+USER user
+
+COPY . .
```

Comment:
Ensure that sensitive files or directories are not copied into the Docker image. You might want to use a `.dockerignore` file to exclude unnecessary files from being copied.